index.js

```diff
@@ -17,6 +17,38 @@ function Changelog( options ) {
 	this.options = options;
 	this.repo = new Repo( this.options.repo );
 
+	var ticketTypes;
+	if ( options.ticketTypes ) {
+		ticketTypes = options.ticketTypes;
+	} else {
+		ticketTypes = [ "github" ];
+	}
+
+	this.ticketTypes = [];
+	// Re-order ticketTypes according to the key order in Changelog.ticketParsers
```

Comment:
It is an option: https://github.com/scottgonzalez/changelogplease/pull/8/files#diff-168726dbe96b3ce427e7fedce31bb0bcR21